test/object-store/schema.cpp

```diff
@@ -1172,7 +1178,15 @@ TEST_CASE("Schema") {
                 {"object", {{"int", PropertyType::Int, Property::IsPrimary{false}, Property::IsIndexed{true}}}}};
             auto object_schema = &*schema1.find("object");
             REQUIRE(schema1.compare(schema2) ==
-                    vec{(AddIndex{object_schema, &object_schema->persisted_properties[0]})});
+                    vec{(AddIndex{object_schema, &object_schema->persisted_properties[0], IndexType::General})});
+        }
+
+        SECTION("add fulltext index") {
```

Comment:
Can a test be added to check for removing a full text index?

Reply:
Yes